lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb

```diff
@@ -204,6 +224,11 @@ def add_file_accessors_paths_to_pods_group(file_accessor_key, group_key = nil, r
 
               pod_name = file_accessor.spec.name
               local = sandbox.local?(pod_name)
+              if keep_folder_structure
+                local = true
```

Comment:
maybe rename the variable to `preserve_file_structure` rather than conflating the idea of what is a local / development pod